extension/src/minn_maxn.rs

```diff
@@ -0,0 +1,1177 @@
+use pgx::{iter::SetOfIterator, *};
+
+use serde::{Deserialize, Serialize};
+
+use crate::{
+    aggregate_utils::in_aggregate_context,
+    flatten,
+    palloc::{Inner, Internal, InternalAsValue, ToInternal},
+    pg_type,
+    raw::bytea,
+    ron_inout_funcs,
+};
+
+use ordered_float::NotNan;
+
+use std::cmp::Reverse;
+use std::collections::BinaryHeap;
+use std::marker::PhantomData;
+
+#[derive(Clone, Serialize, Deserialize)]
+pub struct NMostTransState<T: Ord> {
```

Comment:
This struct should have `Debug` derived as well to make debugging easier

Reply:
Good catch.